pages/_document.js

```diff
@@ -10,8 +10,6 @@ class MyDocument extends Document {
     return (
       <Html>
         <Head>
-          <meta charSet="UTF-8" />
-          <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Any reason why are we removing this ?

Reply:
bcz, this are the duplicate tags. next have this tags already